.gitignore

```diff
@@ -54,3 +54,5 @@ php_test_results_*.txt
 docker-compose.yml
 build/gccarch
 tests/_cache
+.zephir/
+.temp/
```

Comment:
@CameronHall Zephir >= 0.11 uses `.zephir` and Zephir < 0.11` .temp`

Reply:
@sergeyklay I'll remove `.temp` in the next PR.